llvm/lib/CodeGen/WindowScheduler.cpp

```diff
@@ -198,6 +198,12 @@ bool WindowScheduler::initialize() {
     if (MI.isMetaInstruction() || MI.isTerminator())
       continue;
     if (MI.isPHI()) {
+      if (Register AntiReg = getAntiRegister(&MI))
+        // Register with Kernel in phi is not defined within the Kernel itself.
```

Comment:
By kernel do you mean MachineBasicBlock?

Reply:
Updated